src/main/kotlin/no/nav/helsearbeidsgiver/utils/json/SerializationUtils.kt

```diff
@@ -35,3 +37,7 @@ fun LocalDate.toJson(): JsonElement = toJson(LocalDateSerializer)
 fun LocalDateTime.toJson(): JsonElement = toJson(LocalDateTimeSerializer)
 
 fun UUID.toJson(): JsonElement = toJson(UuidSerializer)
+
+fun Orgnr.toJson(): JsonElement = toJson(Orgnr.serializer())
+
+fun Fnr.toJson(): JsonElement = toJson(Fnr.serializer())
```

Comment:
Sniker med disse. Det er bare en shorthand for to verdier som ofte må serialiseres.